nncf/sparsity/schedulers.py

```diff
@@ -150,6 +150,10 @@ def epoch_step(self, next_epoch=None):
     def current_sparsity_level(self):
         if self.sparsity_target_epoch == 0:
             return self.sparsity_target
+
+        if self.current_epoch == -1:
```

Comment:
Should we do the same for other schedulers? Is there a test for this behaviour?

Reply:
Yes, we should. I suggest doing it in a separate pull request. For example, when we will move all schedulers implementation to a common part.

Reply:
Could you create a follow-up ticket for this then?

Reply:
I created a separate story for this task. (Ref: 49837)